app/domains/fluggastrechte/formular/context.ts

```diff
@@ -1,23 +1,23 @@
 import { z } from "zod";
 import { airlineSchema } from "~/services/validation/airline";
 import { airportSchema } from "~/services/validation/airport";
-import { checkedRequired } from "~/services/validation/checkedCheckbox";
 import { optionalOrSchema } from "~/services/validation/optionalOrSchema";
 import { stringOptionalSchema } from "~/services/validation/stringOptional";
 import { fluggastrechteFlugdaten } from "./flugdaten/context";
+import { fluggastrechteGrundvoraussetzungen } from "./grundvoraussetzungen/context";
 import { fluggastrechtePersoenlichDaten } from "./persoenlicheDaten/context";
 import { fluggastrechtProzessfuehrungDaten } from "./prozessfuehrung/context";
 import { fluggastrechtStreitKostenDaten } from "./streitwertKosten/context";
 
 export const fluggastrechtContext = {
-  datenverarbeitungZustimmung: checkedRequired,
   startAirport: optionalOrSchema(airportSchema),
   endAirport: optionalOrSchema(airportSchema),
   fluggesellschaft: optionalOrSchema(airlineSchema),
   ...fluggastrechteFlugdaten,
   ...fluggastrechtePersoenlichDaten,
   ...fluggastrechtStreitKostenDaten,
   ...fluggastrechtProzessfuehrungDaten,
+  ...fluggastrechteGrundvoraussetzungen,
```

Comment:
```suggestion
  ...fluggastrechtGrundvoraussetzungen,
```
we are mixing plural and singular here, I think we should aim for plural like for `fluggastrechteFlugdaten`

Reply:
The one I created it's correct, I will fix the others